python/metatensor-learn/metatensor/learn/selection/feature_selection.py

```diff
@@ -0,0 +1,93 @@
+"""
+Wrappers for the feature selectors of `scikit-matter`_.
+
+.. _`scikit-matter`: https://scikit-matter.readthedocs.io/en/latest/selection.html
+"""
+
+from skmatter._selection import _CUR, _FPS
+
+from ._selection import GreedySelector
+
+
+class FPS(GreedySelector):
+    """
+    Transformer that performs Greedy Feature Selection using Farthest Point Sampling.
+
+    If `n_to_select` is an `int`, all blocks will have this many features selected. In
+    this case, `n_to_select` must be <= than the fewest number of features in any block.
+
+    If `n_to_select` is a dict, it must have keys that are tuples corresponding to the
+    key values of each block. In this case, the values of the `n_to_select` dict can be
+    int that specify different number of features to select for each block.
```

Comment:
Do we really want to use a dict here? Or should we do the same trick as the other classes (i.e. `Labels` + `List[T]`)?

Reply:
and `Labels` are the keys of the blocks? and why a `List[Tensors]` and not `List[int]`?

Reply:
I meant `List[int]`, the `T` above was used as a template parameter!